app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+---
+title: Encapsulated style with Polymer 2.0 and shadow DOM v1
+---
+
+<!-- toc -->
+
+## Styling your elements
+
+Polymer supports DOM templating and the shadow DOM API. When you use the `<dom-module>` and `<template>` blocks to template your custom element, Polymer attaches a shadow root to instances of this element. Polymer then copies in the contents of the template you provided for your element.
```

Comment:
So the kindly folks in core just added a _new_ way to define a DOM template, in addition to the old standard `dom-module`. We talk about that in the DOM templating chapter, so maybe just be less specific here: "When you provide a DOM template for your custom element, ..."